test_data/yam/agama_auto_tumbleweed.yaml

```diff
@@ -1,2 +1,7 @@
 ---
 os_release_name: openSUSE Tumbleweed
+guided_partitioning:
+  disks:
+    - vda
+unused_disks:
+  - vdb
```

Comment:
I have one monitor comment here, I Can see in job setting you have set hdd2 to 60gb, and hdd1 to 40gb, correct?
in that case, I think you may need to set disk to `vdb` and unused_disks to `vda`.

Reply:
From the VR I see that it validates with vda, even if this is the second disk prior to installation. I guess, that the disk numbering after a reboot is done in a way, that the boot disk is always treated as the first disk.

Reply:
Hmm, got it, so the installation seems force changing the device boot order even disk name, is it expected result?
IMO, once we install the system to bigger disk [It shold be `vdb` rather than `vda`] :)

Reply:
The requirement just installed system at biggest disk. We have anther ticket to do installed system at second disk.